docs/glossary.rst

```diff
@@ -0,0 +1,85 @@
+.. Fledge Glossary
+
+.. |github| raw:: html
+
+    <a href="https://github.com/fledge-iot/fledge">Fledge GitHub repository</a>
+
+********
+Glossary
+********
+
+The following are a set of definitions for terms used within the Fledge documentation and code, these are designed to be an aid to understanding some of the principles behind Fledge and improve the comprehension of the documentation by ensuring all readers have a common understanding of the terms used. If you feel any terms are missing or not fully explained please raise an issue against, or contribute to, the documentation in the |github|.
```

Comment:
Yes, it's a bit of a pain, but we will have to remove that sentence in the white labelling. It would be good if there was a conditional mechanism in RST but I don't know of one.

Reply:
AFAIK, there is no such option for a conditional mechanism in RST. The only option we can do with sed command in our customised scripts